guides/get-started-microservices-on-kubernetes.adoc

```diff
@@ -18,14 +18,14 @@ We will start building a link:https://docs.docker.com/[Docker Image, window="_bl
 
 === Guides in this series
 
-* link:simple-microservice-part1[{simple-microservice-part1}]
-* link:simple-microservice-part2[{simple-microservice-part2}]
-* link:simple-microservice-database-part1[{simple-microservice-database-part1}]
-* link:simple-microservice-database-part2[{simple-microservice-database-part2}]
-* link:simple-microservice-infinispan-part1[{simple-microservice-infinispan-part1}]
-* link:simple-microservice-infinispan-part2[{simple-microservice-infinispan-part2}]
-* link:simple-microservice-jms-part1[{simple-microservice-jms-part1}]
-* link:simple-microservice-jms-part2[{simple-microservice-jms-part2}]
+* link:{{ site.baseurl }}/guides/get-started-microservices-on-kubernetes/simple-microservice-part1[{simple-microservice-part1}]
```

Comment:
This will work, but `site.baseurl` is not configured with any value at https://github.com/wildfly/wildfly.org/blob/main/_config.yml#L22, so at the end an `href="/guide..."` is generated.

I think if we should use something here it should be `site.url` instead. However these `{{` `}}` are not too friendly with .adoc, so it is much better if you do not use anything and let the href be specified starting from `/guides`, which indeed is a path from root, so it should do what we need.

I have no idea why Jekyll, when run locally, appends the base URL when you are using relative paths and it does not do it when it is deployed in GitHub.

If this gets merged as it is, it will work, but just because finally it produces an `href=/guides`

Reply:
@yersan Can you send a PR getting rid of the {{ site.baseurl }}? Thanks. I merged this because it worked and the existing links were broken, but longer term it's better not to have misleading stuff in the code.

Reply:
@bstansberry  Donem see https://github.com/wildfly/wildfly.org/pull/651